metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -73,6 +73,27 @@ def make_ml_feature_table_urn(platform: str, feature_table_name: str) -> str:
     )
 
 
+def make_ml_model_urn(platform: str, model_name: str, env: str) -> str:
+
+    return f"urn:li:mlModel:(urn:li:dataPlatform:{platform},{model_name},{env})"
+
+
+def make_s3_urn(s3_uri: str, env: str, suffix: Optional[str] = None) -> str:
```

Comment:
seems odd to have this here - can we move it to one of the aws common files?

Reply:
Yep, will do!